[dynamo] reconstruct functions decorated in the compiled region properly

williamwen42 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #150586
* __->__ #150645

We were previously unable to reconstruct functions that were decorated in the compiled region.


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

williamwen42 committed:
[dynamo] reconstruct functions decorated in the compiled region properly

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] reconstruct functions decorated in the compiled region properly"


We were previously unable to reconstruct functions that were decorated in the compiled region.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] reconstruct functions decorated in the compiled region properly"


We were previously unable to reconstruct functions that were decorated in the compiled region.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

jansel reviewed (changes requested):
Is that compile time regression real?

williamwen42 commented:
@pytorchbot rebase

williamwen42 committed:
Update

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] reconstruct functions decorated in the compiled region properly"


We were previously unable to reconstruct functions that were decorated in the compiled region.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 committed:
Update on "[dynamo] reconstruct functions decorated in the compiled region properly"


We were previously unable to reconstruct functions that were decorated in the compiled region.


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 commented:
@pytorchbot merge
